parameterspace/configspace_utils.py

```diff
@@ -0,0 +1,132 @@
+"""Initialize a `ParameterSpace` from a `ConfigSpace` JSON dictionary."""
+
+from typing import List, Optional
+
+import parameterspace as ps
+from parameterspace.condition import Condition
+from parameterspace.utils import verify_lambda
+
+
+def _escape_parameter_name(name: str) -> str:
+    """Replace colons with underscores.
+
+    Colons are incompatible as ParameterSpace parameter names.
+    """
+    return name.replace(":", "_")
+
+
+def _get_condition(conditions: List[dict], parameter_name: str) -> Optional[Condition]:
+    """Construct a lambda function that can be used as a ParameterSpace condition from a
+    ConfigSpace conditions list given a specific target parameter name.
+    """
+    condition = Condition()
+
+    varnames = []
+    function_texts = []
+    for cond in conditions:
+        if cond["child"] == parameter_name:
+            parent = _escape_parameter_name(cond["parent"])
+            varnames.append(parent)
+            # The representation is used because it quotes strings.
+            if cond["type"] == "IN":
+                # Using the valid Python "a in [2,3,4]" is not supported, hence use or
```

Comment:
I am surprised by this comment. Does that refer to the way the lambda function is constructed here, because parameterspace does support the pythonic `a in [2,3,4]` as a condition?

Reply:
IIRC we don't treat this as a safe statement for de-/serialization, which makes the explicitly constructed or equal statement more useful. I'd be happy to spend some time on making `in` safe for de-serialization, though.

Reply:
Should be resolved with #50, on which I'll rebase once merged and simplify the condition here.